scalajvm/app/controllers/Quotes.scala

```diff
@@ -23,11 +25,10 @@ object Quotes extends Controller {
     Ok(views.html.index(quotes, pageNumber, countPages, order, filter))
   }
 
-  def getRSSFeed = ActionWithTx { implicit request =>
+  def getRSSFeed = (new ActionWithTx(cc)) { implicit request =>
     import request.dbSession
-    import play.api.Play.current
 
-    val feedLimit = current.configuration.getInt("feed.limit").get
+    val feedLimit = configuration.get[Int]("feed.limit")
```

Comment:
Just plain `.get`? What if no "feed.limit" will be present in configuration?

Reply:
Then bad things will happen :(

I realize there's a problem, but I don't intend to fix it in this PR because this one is about getting stuff to compile and run again. If you feel strongly enough about this, please file an issue, and we'll get to it sometime (unless we decide to move off Scala entirely).

Reply:
This is fine.